### ISSUE TITLE
The model needs to face positive Z axis - error

### ISSUE THREAD

**Issue by 3dcinetv** [esperecyan/UniVRMExtensions#7]
I used Blender to export the model (I've been exporting .fbx from Blender for 7 years without issue, including Blender 3.2).
When I added the UniVRMExtensions package in Unity 2019.4.29 to create my VRM avatar (after normalizing and spring/collider scripts):
"The model needs to Face positive Z axis" error.
![image](https://user-images.githubusercontent.com/11066585/208887716-6809ff72-bf70-4c41-89f4-7d5e25d83e8b.png)

Further, the -Z axis is set to Blender by Default, and I've never gotten this issue before. 
When I export .fbx with Z (positive), Unity gets stuck with the inspector showing this (and I cannot finish configuring my avatar):

Please help. This is a RECENT error in the update of UniVRMExtensions?
BTW: I use UniVRMExtensions alone, because it installs the VRM exporter, and causes conflict if I install the default one.

**Comment by 3dcinetv** [comment]
My workflow is: 
1. Export from Blender, the model rigged.
2. Import to Unity, assign materials, assign avatar mecanim, configure blendshapes legacy import
3. Place the .fbx model into the scene and export it using UniVrmExtensions>Export .VRM to NORMALIZE the VRM.
4. I configure the blendshapes and scripts (springs and colliders) and export using UniVrmExtensions>Export .VRM 
5. and then I get the "model needs to face Z forward". So it it clear that after NORMALIZING the vrm, UniVrmExtensions>Export .VRM causes this issue.
6. I also enforced the T pose before exporting which gives the error.

**Comment by esperecyan** [comment]
Are you saying that if you directly “VRM0 → Export to VRM 0.x” an FBX, the problem does not occur, but if you “VRM0 → プレハブバリアントを作ってVRMプレハブ化” and then “VRM0 → Export to VRM 0.x” the prefab, the problem occurs?

**Comment by 3dcinetv** [comment]
Yes. Re-exporting a prefab will yield a stange error "The character must face positive Z axis".
So I did a huge workaround:
I created a new Scene within the same project. I took the Normalized VRM model, and started adding the spring scripts. I noticed I could not export (even with 1 spring script) because one of the root numbers for the spring was empty. I deleted that empty name field, and the UniVrmExtensions>Export VRM allowed me to export. 
I added all the other collisions and springs checking not a single one has empty root names in the script. 
I finally was able to export this way.
